pkg/cmd/pulumi/convert.go

```diff
@@ -100,7 +100,15 @@ func newConvertCmd() *cobra.Command {
 		Short: "Convert Pulumi programs from a supported source program into other supported languages",
 		Long: "Convert Pulumi programs from a supported source program into other supported languages.\n" +
 			"\n" +
-			"The source program to convert will default to the current working directory.\n",
+			"The source program to convert will default to the current working directory.\n" +
+			"\n" +
+			"Valid source languages: yaml, terraform\n" +
```

Comment:
Can we add something to make it clear these aren't the _only_ valid source languages. Zaid's written converters for ARM and Bicep as well, and I suspect we'll do CloudFormation and k8s manifests at some point as well.

Reply:
I was under the impression that these were the only valid source languages. Definitely happy to update with the complete list!

Just to verify, the valid source languages at the moment are:

-  yaml
- terraform
- arm
- bicep

Is this correct @Frassle?

Reply:
Yup that's correct today, but as it's a plugin system so anyone (even third parties) could add more. That's why as well as updating the list, I'd also rephrase it a bit to make it clear that the list is not necessarily exhaustive.

Reply:
Pushing back a bit here because I worry about users being confused if we tell them that this isn't a comprehensive list because it gives the impression that there are more languages that we are just not including in the docs for some reason. I think it makes more sense to just update the list whenever a new language is actively supported .

Reply:
> The convert system is a plugin system. Anyone can write a convert plugin and thus add a new language to the list of what's valid for the "--from" parameter, they're not all "supported" by us but they're still valid options.

Will brainstorm with the team to see how to best accommodate for this.

Reply:
Will we ever automatically download a third-party plugin? If not, then quoting the full set of the sources we support (and automatically download for) here would make sense to me, along with some mention of a doc (which I don't think yet exists) that lists additional (perhaps third-party) plugins along with instructions for how to create them.

Reply:
>Will we ever automatically download a third-party plugin?

Yes. Currently we manually add some third-party plugins sources to the CLI so it can download @Zaid-Ajaj's bicep and arm converters without needing the --server option. We might add others there as well.

Long term I suspect we'd probably have some way to put converter plugins on the registry and we'd look them up from there. But at that point we could probably also ask registry for the list of all possible plugins as well.